proto/neutron/dex/deposit_record.proto

```diff
@@ -18,4 +19,11 @@ message DepositRecord {
   int64 lower_tick_index = 4;
   int64 upper_tick_index = 5;
   uint64 fee = 6;
+  string total_shares = 7 [
+    (gogoproto.moretags) = "yaml:\"total_shares\"",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
```

Comment:
`github.com/cosmos/cosmos-sdk/types.Int` is deprecated and it is an alias to `cosmossdk.io/math.Int`. But it's not necessary to fix it right now.  We want to move sometime, but in standalone task